pype/lib/applications.py

```diff
@@ -213,7 +212,7 @@ class ApplicationAction(avalon.api.Action):
     @property
     def log(self):
         if self._log is None:
-            self._log = Logger().get_logger(self.__class__.__name__)
+            self._log = logging.getLogger(self.__class__.__name__)
```

Comment:
`ApplicationAction` class will be removed from here so it doesn't matter byt why changed from `Logger().get_logger` to `logging.getLogger`?